### PR TITLE
rustc option "instrument-coverage" stabilized

### PR DESCRIPTION
**Pull request by kazuk**
then change `-Z` to `-C`

see: https://github.com/rust-lang/rust/pull/90132

### CODE REVIEW

**Comment by marco-c** [comment]
@kazuk it is not stable yet in the latest Rust version. We need to wait on 1.59.

**Comment by kazuk** [comment]
My understanding, grcov requires "Nightly" Rust.
So I think no need to wait "Stable" Rust 1.59.

**Comment by marco-c** [comment]
It is possible to use stable rust (by setting the RUSTC_BOOTSTRAP environment variable).

**Comment by kazuk** [comment]
I update Usage line, and update "grcov with Travis" example.

File: README.md [hunk 1]
```diff
@@ -173,7 +172,7 @@ RUSTC_BOOTSTRAP=1`, which basically turns your stable rustc into a Nightly one.
 2. Ensure that the following environment variable is set up:
 
    ```sh
-   export RUSTFLAGS="-Zinstrument-coverage"
+   export RUSTFLAGS="-C instrument-coverage"
```
Comment (marco-c) [inline README.md:175]:
```suggestion
   export RUSTFLAGS="-Cinstrument-coverage"
```

**Review by marco-c — APPROVED** [review]
Thanks!

**Pull request merged**